servor.js

```diff
@@ -129,15 +129,11 @@ http
 // ----------------------------------
 // Get available IP addresses
 // ----------------------------------
-const ips = []
 const interfaces = require('os').networkInterfaces()
-for (const interface in interfaces) {
-  interfaces[interface].forEach(ip => {
-    if (ip.family === 'IPv4' && ip.internal === false) {
-      ips.push(ip.address)
-    }
-  })
-}
+const ips = Object.values(interfaces)
+  .flat()
```

Comment:
Woahhh, did not know about `flat` (or at least I haven't used it in the wild). That is awesome!